addons/account_payment/wizards/payment_link_wizard.py

```diff
@@ -1,11 +1,40 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from werkzeug import urls
+
+from odoo import _, api, fields, models
+from odoo.addons.payment import utils as payment_utils
+from odoo.tools import format_date, formatLang
 
 
 class PaymentLinkWizard(models.TransientModel):
     _inherit = 'payment.link.wizard'
 
+    open_installments = fields.Binary(export_string_translation=False)
+    open_installments_preview = fields.Html(
+        export_string_translation=False, compute='_compute_open_installments_preview'
+    )
+
+    @api.depends('open_installments')
+    def _compute_open_installments_preview(self):
+        for wizard in self:
+            preview = ""
+            for installment in wizard.open_installments:
```

Comment:
That's not a relational field :raised_eyebrow:

Reply:
Resolved with https://github.com/odoo/odoo/pull/174812/commits/4696befbdb59123520af1e00b1da7d0a2605ce99#diff-4e23ba6a95cdda49ecc9b74a6ac55a85237c56e2e31afe0f8a4906a5827ff39cR17